From 65df0c1e556bfcbb39382da267141676a894decd Mon Sep 17 00:00:00 2001
From: Hongbo Zheng <zhenghongbo3@huawei.com>
Date: Wed, 31 Mar 2021 18:01:44 +0800
Subject: [PATCH 086/189] net/hns3: fix code style

Add one space before the left brace to solve the static warning.

Fixes: 63e05f19b82b ("net/hns3: support Rx descriptor status query")

Signed-off-by: Hongbo Zheng <zhenghongbo3@huawei.com>
Signed-off-by: Min Hu (Connor) <humin29@huawei.com>
---
 drivers/net/hns3/hns3_rxtx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/hns3/hns3_rxtx.c b/drivers/net/hns3/hns3_rxtx.c
index c41cccb..b06b723 100644
--- a/drivers/net/hns3/hns3_rxtx.c
+++ b/drivers/net/hns3/hns3_rxtx.c
@@ -4477,7 +4477,7 @@ hns3_dev_rx_descriptor_status(void *rx_queue, uint16_t offset)
 		if (offset >= rxq->nb_rx_desc - rxq->rx_free_hold)
 			return RTE_ETH_RX_DESC_UNAVAIL;
 	} else if (dev->rx_pkt_burst == hns3_recv_pkts_vec ||
-		   dev->rx_pkt_burst == hns3_recv_pkts_vec_sve){
+		   dev->rx_pkt_burst == hns3_recv_pkts_vec_sve) {
 		if (offset >= rxq->nb_rx_desc - rxq->rx_rearm_nb)
 			return RTE_ETH_RX_DESC_UNAVAIL;
 	} else {
-- 
2.7.4

